tests/snappi_tests/multidut/pfc/files/lossless_response_to_external_pause_storms_helper.py

```diff
@@ -126,6 +127,21 @@ def run_lossless_response_to_external_pause_storms_test(api,
                                                    all_flow_names=all_flow_names,
                                                    exp_dur_sec=DATA_FLOW_DURATION_SEC + DATA_FLOW_DELAY_SEC,
                                                    snappi_extra_params=snappi_extra_params)
+    interface_stats = get_interface_stats
+    tx_port1 = tx_port[0]['peer_port']
+    tx_port2 = tx_port[1]['peer_port']
+    # Fetch relevant statistics
+    pkt_drop1 = interface_stats(ingress_duthost, tx_port1)[ingress_duthost.hostname][tx_port1]['rx_fail']
+    pkt_drop2 = interface_stats(ingress_duthost, tx_port2)[ingress_duthost.hostname][tx_port2]['rx_fail']
+    rx_pkts_1 = interface_stats(ingress_duthost, tx_port1)[ingress_duthost.hostname][tx_port1]['rx_pkts']
```

Comment:
Is there a reason why we create a seperate function variable and then use it instead of directly using the function name ? I am curious.